paparazzi-gradle-plugin/src/test/projects/layout-direction-compose/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
```

Comment:
nice tests! 

some feedback:
1) eliminate one of the compose/xml versions.  since this is about resource loading, having two plugin tests is duplicative.  I'd vote for xml, since it's simple/faster on CI, but it doesn't really matter.
2) let's use TestParameterInject here instead of unsafeUpdateConfig.  Would clean up the test a lot.
3) reduce from 4 locales to 2, and maybe move ar to layoutDirection, since it's a better example of RTL layouts, vs en_gb.